### PR TITLE
ENH Incorporate upstream CUDA 11+ version changes

### PR DESCRIPTION
**Pull request by mike-wendt**
Recently `nvidia/cuda` has changed it's [tags]() to include the patch version as well. To pull CUDA 11.2 and newer images we need to update our builds to use this patch version

While `nvidia/cuda` is incorporating the patch version into tags, our plan is to not incorporate them in our images to maintain compatibility with existing scripts and services

### CODE REVIEW

**Comment by mike-wendt** [comment]
Looks like CentOS 7 builds for CUDA 11.2.2 are blocked by this [issue](https://gitlab.com/nvidia/container-images/cuda/-/issues/115)

I reached out to say I've seen the same issue previously and it appears to be a false positive. Will add an exception to the axis for now to continue with testing.

**Comment by mike-wendt** [comment]
In order to get rapidsai/integration#232 to where I can test for conflicts in package builds I need to merge this PR to get `miniconda-cuda-driver` images out

There will need to be follow-up PRs to this one:
- Restoring CentOS 7 builds for 11.2.2 when they are available
- `rapidsai` and `rapidsai-driver` image updates once rapidsai/integration#232 is merged

**Review by raydouglass — APPROVED** [review]

**Review by ajschmidt8 — APPROVED** [review]

**Pull request merged**
